src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/messaging/PrisonerReceivedIntoPrisonEventService.kt

```diff
@@ -40,16 +40,19 @@ class PrisonerReceivedIntoPrisonEventService(
       prisonAdmittedTo = prisonId,
       eventDate = eventOccurredAt,
     )
+
+    /*
+    TODO - replace the above call to the CiagKpiService with the following logic / calls to the InductionScheduleService and ReviewScheduleService
+       - If prisoner has no Induction, Action Plan or Induction Schedule, then create the Induction Schedule
+       - If prisoner has an Induction Schedule that is complete, create or update their Review Schedule with status SCHEDULED, reason PRISONER_READMISSION and due date +10 days
+       - if prisoner has an Induction Schedule, and it is not complete, reschedule it with the same deadline date rules as if it were a new admission (even if it was exempt (exempt thru transfers handled below))
+     (check all of the above rules with Aliki)
+     */
   }
 
   private fun PrisonerReceivedAdditionalInformation.processPrisonerTransferEvent() {
     log.info { "Processing Prisoner Admission Event (due to transfer) for prisoner [$nomsNumber]" }
 
-    ciagKpiService?.processPrisonerTransfer(
```

Comment:
The implementation did nothing, and I've added the TODO with jira reference to implement the correct behaviour (likely via `InductionScheduleService` method calls)